x-pack/packages/kbn-ai-assistant/src/hooks/use_local_storage.test.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useLocalStorage } from './use_local_storage';
+
+describe('useLocalStorage', () => {
```

Comment:
I don't have a good answer for this but it kind of sucks seeing these files duplicated. Some of it was moved to `@kbn/observability-utils` but I can see how that cannot be used in the Search AI Assistant.